plugin/storage/badger/docs/storage-file-non-root-permission.md

```diff
@@ -0,0 +1,46 @@
+# Badger file permissions as non-root service
+
+After the release of 1.50, Jaeger's Docker image is no longer running with root privileges (in [#4783](https://github.com/jaegertracing/jaeger/pull/4783)), and issues such as "permission denied" occur when writing.
+
+If you encounter this type of problem, you can update the owner of the data file before starting Jaeger.
+
+Here is a sample `docker-compose.yml`:
+
+```yaml
+version: "3.9"
+
+services:
+[...]
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    command:
+      - "--badger.ephemeral=false"
+      - "--badger.directory-key=/badger/data/keys"
+      - "--badger.directory-value=/badger/data/values"
+      - "--badger.span-store-ttl=72h0m0s" # limit storage to 72hrs
+    environment:
+      - SPAN_STORAGE_TYPE=badger
+    volumes:
```

Comment:
```suggestion
    # Mount host directory "jaeger_badger_data" as "/badger" inside the container.
    # The actual data directory will be "/badger/data", 
    # since we cannot change permissions on the mount.
    volumes:
```